Rsk.Samples.IdentityServer.AdminUIIntegration.Tests/CustomSamlIdentityProviderStoreTests.cs

```diff
@@ -0,0 +1,113 @@
+                "\"NameIdClaimType\":\"http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier\"," +
+                "\"SignedOutCallbackPath\":\"\"," +
+                "\"IdPInitiatedSsoCompletionPath\":\"\"," +
+                "\"AllowIdpInitiatedSso\":false," +
+                "\"SkipUnrecognizedRequests\":false," +
+                "\"RequireSamlResponseDestination\":true," +
+                "\"RequireSamlMessageDestination\":true," +
+                "\"RequireSignedLogoutRequests\":false," +
+                "\"RequireSignedLogoutResponses\":false," +
+                "\"RequireAuthenticatedUserForLogoutRequests\":false," +
+                "\"RequireSignedArtifactResponses\":true," +
+                "\"RequireSignedArtifactResolveRequests\":true," +
+                "\"AllowedIdpInitiatedRelayStates\":[]," +
+                "\"ThrowOnLogoutErrors\":true," +
+                "\"SamlCspOptions\":" +
+                    "{\"Level\":1," +
+                    "\"AddDeprecatedHeader\":true}," +
+                "\"ArtifactDeliveryBindingType\":\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect\"," +
+                "\"ArtifactLifeTime\":\"00:05:00\"," +
+                "\"ArtifactResolutionService\":\"\"," +
+                "\"SigningOptions\":" +
+                    "{\"CanonicalizationMethod\":\"http://www.w3.org/2001/10/xml-exc-c14n#\"," +
+                    "\"DigestAlgorithm\":\"http://www.w3.org/2001/04/xmlenc#sha256\"," +
+                    "\"SignatureAlgorithm\":\"http://www.w3.org/2001/04/xmldsig-more#rsa-sha256\"}," +
+                "\"LogSamlMessages\":false," +
+                "\"SkipAuthnContextCheck\":false," +
+                "\"CallbackPath\":\"/callbackPath\"," +
+                "\"SignInScheme\":\"SignInScheme\"," +
+                "\"SaveTokens\":false," +
+                "\"RemoteAuthenticationTimeout\":\"00:15:00\"," +
+                "\"AccessDeniedPath\":\"\"," +
+                "\"BackchannelTimeout\":\"00:01:00\"," +
+                "\"ReturnUrlParameter\":\"ReturnUrl\"}"
+        };
+        var sut = CreateSut();
+
+        var result = sut.MapIdp(identityProviderEfEntity);
+
+        Assert.NotNull(result);
+        Assert.Equal(result.Scheme, identityProviderEfEntity.Scheme);
+        Assert.Equal(result.DisplayName, identityProviderEfEntity.DisplayName);
+        Assert.Equal(result.Type, identityProviderEfEntity.Type);
+        Assert.Equal(result.Enabled, identityProviderEfEntity.Enabled);
+
+        var samlOptions = new SamlDynamicIdentityProvider(result).SamlAuthenticationOptions;
+        
+        var optionsToString = JsonConvert.SerializeObject(samlOptions, SamlDynamicIdentityProvider.SerializerSettings);
+        
+        Assert.Equal(optionsToString, identityProviderEfEntity.Properties);
+        
+        // Assert.Equal(samlOptions.ServiceProviderOptions.EntityId, SpEntityId);
```

Comment:
Prefer not see commented out code, makes you wonder if it's been commented out by mistake?

Reply:
Done. Thanks :)